qiwis.py

```diff
@@ -213,36 +216,29 @@ def __init__(
         """
         super().__init__(parent=parent)
         self.appInfos: Dict[str, AppInfo] = {}
-        icon_path, background_path = map(
-            lambda attr: getattr(constants, attr, None),
-            ("icon_path", "background_path")
+        icon_path, background_path, background_color = map(
+            lambda name, default: getattr(constants, name, default),
+            (("icon_path", ""), ("background_path", ""), ("background_color", "ffffff"))

```

Comment:
Each tuple, e.g., ("icon_path", ""), is given as `name`, which raises the error that `default` is not given.
How about just using comprehension?
```python
... = (
  getattr(constants, name, default) for name, default in
  (("icon_path", ""), ("background_path", ""), ("backgroun_color", "ffffff"))
)
```